direct/src/showbase/ShowBase.py

```diff
@@ -382,7 +382,7 @@ def __init__(self, fStartDirect: bool = True, windowType: str | None = None) ->
         #: yourself every frame.
         self.cTrav: CollisionTraverser | Literal[0] = 0
         self.shadowTrav: CollisionTraverser | Literal[0] = 0
-        self.cTravStack = Stack()
+        self.cTravStack = Stack[CollisionTraverser]()
```

Comment:
How does this work? Does Stack implement `__getitem__`? Should this not just be an annotation?